extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js

```diff
@@ -152,6 +156,39 @@ export class AmpAdNetworkDoubleclickImpl extends AmpA4A {
     ]);
   }
 
+  initializeRefreshIfEligible() {
+    const refreshConfig = this.getRefreshConfiguration();
+    if (!refreshConfig) {
+      return;
+    }
+    new RefreshManager(this.win, this.element, refresher => {
+      this.isRefreshing = true;
+      this.tearDownSlot();
```

Comment:
Seems like this logic should be within AmpA4A.  I realize it means we'll bloat for networks that do not support refresh but it seems to me to be a cleaner way to ensure proper integration (no need to expose getAdPromise for instance).   Assume that refreshmanager will call a function on the A4A instance (e.g. refresh())).  Advantage is that at least refresh manager is not within AmpA4A and so it won't be compiled into networks that don't need it.  Also, make getRefreshConfiguration part of RefreshManager.